server/sonar-web/src/main/js/apps/coding-rules/templates/rule/coding-rules-profile-activation.hbs

```diff
@@ -7,7 +7,7 @@
     {{/if}}
   </div>
 
-  <div class="modal-body modal-container">
```

Comment:
`.modal-container` is required to make sure the modal content fits on the screen. Try to add 10 form fields in this form, and you'll see the problem 😉

Reply:
Ok I didn't notice that we could have some parameters field, then the scrollbar should not be removed. I'll revert that change.